### DISCUSSION TITLE
IPv6 tracker connections failing

### DISCUSSION THREAD

**Discussion by mongotron** [transmission/transmission#6380] [Support]
Edit: I have edited this post to reflect what seems to be the true nature of my issue, I originally thought this affected all public trackers but it seems to be more specific to IPv6 connections and/or magnet links.

I have Transmission bound to a port between 1024-65534 which is randomly determined by my VPN provider and changes each time the VPN connection goes up or down. A custom script I wrote sets this port as the only port allowed through my firewall for Transmission each time it changes. When I check the port in Transmission the port is open.

Most of my torrents are through a private tracker and these are working fine, both downloading and seeding. However any torrent I try to add from a public tracker using a magnet link never starts - the status remains "retrieving metadata (0.0%)" and goes nowhere. In transmission.log I see a number of entries similar to this:

```
[2023-12-15 16:37:04.779] WRN tr-udp.cc:265 Couldn't send to [2a02:2788:1208:2f0:d198:5596:1a01:7b8e]:1449: 101 (Network is unreachable) (tr-udp.cc:265)
[2023-12-15 16:37:06.675] WRN tr-udp.cc:265 Couldn't send to [2001:999:485:b4d1:7b6f:880d:8848:b5cb]:35800: 101 (Network is unreachable) (tr-udp.cc:265)
[2023-12-15 16:37:06.675] WRN tr-udp.cc:265 Too many messages like this! I won't log this message anymore this session. (tr-udp.cc:265)
```

I presume these log entries are Transmission trying to send traffic out to an IPv6 address via port 1449 or 35800, which my firewall doesn't allow, and no the torrents can't start. Is that correct? If so, why isn't Transmission using the port it is bound to - is that port for incoming connections only, not outgoing connections? If the outgoing port is randomized is there any way I can set (bind?) the outgoing port for Transmission to use?

If I've got all that wrong, any further help would be appreciated :)

**Comment by GaryElshaw** [comment]
Probably a silly question, but have you disabled IPv6 on your router (if your isp doesnt support it), and in the transmission config file?

**Reply by mongotron** [reply]
Not a silly question, but I'm not seeing an option to disable IPv6 in my config file?

Edit: just to clarify, my firewall won't allow traffic on the ports listed in the log file, but it will allow IPv6 traffic.

**Reply by mongotron** [reply]
I suspect you were referring to the bind-address-ipv6 value? I have this set to "::" by default but I have also tried "fe80::", which I think is meant to 'disable' IPv6 but has no effect on this issue.
